circu_day1.py

```diff
@@ -30,6 +28,8 @@
     if s[i] == "q":
         print("q found at positon: " + str(i))
 
+print("n found at: " + str(s.find("n")))
```

Comment:
Mai clean este să ne folosim de cast la str și concatenare pe care le face oricum print:
```suggestion
print("n found at:", s.find("n"))
```
sau f-strings:
```suggestion
print(f"n found at: {s.find('n')}")
```